app/views/landing.html

```diff
@@ -73,6 +73,10 @@ <h1 class="heading">Scenic Subscription offer</h1>
       // Start subscription flow with the target SKU.
       subscriptions.subscribe('subscription_daily');
     });
+    if ('<% logImpressions %>') {
+      // Configure Analytics using user impressions
+      subscriptions.configure({analyticsMode: AnalyticsMode.IMPRESSIONS});
```

Comment:
We can't use enums outside of our code - this is just a closure thing. So, it should be  just `{analyticsMode: 1}`